### PR TITLE
Update whatsnew: technical novelties

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/whatsnew.rst [hunk 1]
```diff
@@ -93,82 +93,56 @@ space's name wiki".
...
 
-**Upgrades**
+**New features**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In the version 5.1, we integrated a new feature which is the push notifications on mible.
+This new feature is supported by eXo mobile applications. It relies on Firebase Cloud Messaging 
+and a new addon packaged by default in the Enterprise Edition (exo-push-notifications).
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: I don't think we should talk about Push Notifications in technical new features, it is already listed in functional new features.

File: docs/whatsnew.rst [hunk 2]
```diff
@@ -93,82 +93,56 @@ space's name wiki".
...
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
 
-- Elasticseach 5.6 
+- Benefit from its lastest improvements such as:
 
-- Infinispan 8
+  - Use the new security framework WildFly Elytron
 
-- Groovy 2.4
+  - Support for HTTP/2
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: No need to list JBoss EAP improvements IMO. We can just add a link to https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71 .

File: docs/whatsnew.rst [hunk 3]
```diff
@@ -93,82 +93,56 @@ space's name wiki".
...
-  end of indexation.
-
-- An Mbean is added to reload Javascript.
-
-- A new minify attribute allowing to enable/disable the minification.
-
-- Compilation of Javascript scripts by Google Closure Compiler to ES5.
+As cited in the previous section, eXo Chat has a new design. 
+This new design is powered by the frontend framework Vue.js.
+Following this framework change, Chat favorites data has also been changed. An upgrade plugin runs
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: The Chat favorites data model change is not directly related to the new frontend framework. We can talk of both topics (frontend framework change, and favorites migration), but without linking them together.

File: docs/whatsnew.rst [hunk 4]
```diff
@@ -93,82 +93,56 @@ space's name wiki".
...
-
-- Compilation of Javascript scripts by Google Closure Compiler to ES5.
+As cited in the previous section, eXo Chat has a new design. 
+This new design is powered by the frontend framework Vue.js.
+Following this framework change, Chat favorites data has also been changed. An upgrade plugin runs 
+on eXo Platform 5.1 startup to migrate existing data to the new format.
 
-- Improve the Upgrade plugin framework: Add configuration to define the 
-  target version.
+**Some other Improvements**
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: other -> others

File: docs/whatsnew.rst [hunk 5]
```diff
@@ -93,82 +93,56 @@ space's name wiki".
...
 
-- Add the possibility to define a group of users able to manage all 
-  spaces.
+- Harmonize cache configuration in cluster and local modes.
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: We must point to the doc that will be done in https://jira.exoplatform.org/browse/DOC-4214

File: docs/whatsnew.rst [hunk 6]
```diff
@@ -95,80 +95,47 @@ Technical novelties in eXo Platform 5.1
 
 **Upgrades**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: I would rephrase to "eXo Platform 5.1 now supports JBoss EAP 7.1. This new version comes with some improvements and fixes some issues which impacted eXo Platform:
* Remember me" token not recognized in Jboss EAP 7.0 (PLF-7787)
* Instance id of a cluster node is not added on the JSESSIONID cookie (PLF-7854)
"
And set links on :
* "some improvements" -> https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71
* "PLF-7787" -> https://jira.exoplatform.org/browse/PLF-7787
* "PLF-7854" -> https://jira.exoplatform.org/browse/PLF-7854

WDYT ?

File: docs/whatsnew.rst [hunk 7]
```diff
@@ -95,80 +95,47 @@ Technical novelties in eXo Platform 5.1
...
-storage.  
-
-- Cache strategy and configuration
-
-- New cache on organization service
-
-- Files indexing in Elasticsearch
-
-- Storage of all spaces members in Social database tables
+In eXo Platform 5.1 we continue our effort on improving the performance and the scalability
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: continue -> continued

File: docs/whatsnew.rst [hunk 8]
```diff
@@ -95,80 +95,47 @@ Technical novelties in eXo Platform 5.1
...
-
-- When indexing JCR workspace, old indexes are always existing until the 
-  end of indexation.
-
-- An Mbean is added to reload Javascript.
-
-- A new minify attribute allowing to enable/disable the minification.
+-  As cited in the previous section, eXo Chat has a new design. 
+   This new design is powered by the frontend framework Vue.js instead of WebUI, JQuery and Juzu 
+   which are now considered as obselete and not enough productive.
```
Comment (thomasdelhomenie) [inline docs/whatsnew.rst]: I don't think we should write this. It is very negative and most of the platform still uses these technologies. I am not sure even sure it makes sense to talk about vue.js.
But if you want to talk about it, I would just say "As cited in the previous section, eXo Chat has a new design. Moreover, from a technical point of view, it has been implemented using a new frontend framework (Vue.js)."
WDYT ?

File: docs/whatsnew.rst (hunk 6 above)
Comment (mzorai) [reply docs/whatsnew.rst]: OK for the phrase but not really with putting jira's links because sometime there are internal links that may make the full sense incomplete when it is an answer to a previous comment

File: docs/whatsnew.rst (hunk 8 above)
Comment (mzorai) [reply docs/whatsnew.rst]: OK

File: docs/whatsnew.rst [hunk 9]
```diff
@@ -95,80 +95,48 @@ Technical novelties in eXo Platform 5.1
...
-
-- Files indexing in Elasticsearch
-
-- Storage of all spaces members in Social database tables
-
+In eXo Platform 5.1 we continued our effort on improving the performance 
+and the scalability by changing some strategies, configurations and data 
+storage:
+
+-  Users/Groups/Roles import in database: a new job is implmentd to
```
Comment (plamarque) [inline docs/whatsnew.rst]: implement**e**d

File: docs/whatsnew.rst (hunk 6 above)
Comment (thomasdelhomenie) [reply docs/whatsnew.rst]: The goal is to give more information about the issues fixed by this version. Putting only the short summary is not enough for the customer to know if it is the issue he face IMO.

Comment (mzorai) [reply docs/whatsnew.rst]: OK

**Comment by mzorai** [comment]
After discussing with @thomasdelhomenie  we agreed to add the links to issues and the whole content is OK
